src/views/Main.js

```diff
@@ -55,38 +90,40 @@ const Main = props => {
   }
   return (
     <Container>
-      <Button onPress={() => props.navigation.navigate('Test')}>
-        <Text>Test navigation!</Text>
-      </Button>
-      <Content style={styles.content}>
+      <Image style={styles.headerImage} source={require('../../assets/nwMenuHeader.png')}/>
+      <View>
         <H3 style={styles.text}>
-          This is an NFC app and I'm going SQL inject your NFC tag
+          Home
         </H3>
-        {getNFC ? null : (
-          <Text style={styles.text}>
-            {getNFC ? 'NFC Enabled!' : 'NFC not supported.'}
-          </Text>
-        )}
-        <Button style={styles.button} onPress={_read}>
-          <Text>Read</Text>
-        </Button>
-        <Button style={styles.button} onPress={_write}>
-          <Text>Write</Text>
-        </Button>
-        <Button style={styles.button} onPress={_generateUUID}>
-          <Text>Generate UUID: {currUuid}</Text>
-        </Button>
-        <H1 style={styles.text}>{text}</H1>
-        <Button onPress={logout}>
-          <Text>logout!</Text>
-        </Button>
-      </Content>
+        <View>
+            <Image style={styles.button} source={require('../../assets/ButtonBg.png')}/>
```

Comment:
Idk how I feel about this being an image instead of a gradient. If its too much work to make a gradient I suppose we can leave it

But maybe we should make the green button a new component? Looks like a lot of pages use it

Reply:
I tried to refactor the button into a component previously and failed to deal with navigation problems. I gave it a shot again and I found a fix! Thanks. Green button is now a component with two props to pass down: location to navigate to, text.

Reply:
Dope! Can you make it take a callback instead of location? Some buttons need to do things that aren't navigation